### PR TITLE
adds contributing.md file 

### PR DESCRIPTION
**Pull request by yati1998**
Fixes: #12

### CODE REVIEW

File: CONTIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Contributing to Sonores
+Thanks for taking interest in Sonores. The main aim of this project is to translate the labels of a form from English to Hindi and also enable text-to-speech and speech-to-text feature using specific API's.
```
Comment (manaswinidas) [inline CONTIBUTING.md]: API's -> APIs

File: CONTIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+git checkout -b <branch-name>
+```
+- Make necessary changes according to the issue.
+
+- Once you have made the changes push it to the repo using:
+```git
+1. git add .
+2. git commit -m "commit message".
+3. git push <remote> <branch>
+```
```
Comment (manaswinidas) [inline CONTIBUTING.md]: please include the instructions of rebasing and then creating PRs, to pull in the latest changes before one creates a branch for a new issue

Comment (manaswinidas) [reply CONTIBUTING.md]: also include where the changes can be seen, once they ake any changes, like making changes and then reloading the extension in their local chrome://extensions page

File: CONTIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- While writing the commit message , one must follow the [git commit message guidelines.](https://udacity.github.io/git-styleguide/)
+
+- Please create a Pull Request from GitHub to Sonores: master. Please provide a very brief Title and description of PR. Link the PR to issue by adding #issueNumber at the end of the description.
```
Comment (rkpattnaik780) [inline CONTIBUTING.md]: The PR should be raised to the develop branch. Name of repo is Sonores-project, not Sonores.

File: CONTIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+- After you have created the Pull Request, your PR will be reviewed soon.
```
Comment (rkpattnaik780) [inline CONTIBUTING.md]: This sentence looks bit odd.

**Pull request closed without merging**